### DISCUSSION TITLE
JSON to map Bin Type for Colour and Icon

### DISCUSSION THREAD

**Discussion by ciderwarehouse1** [robbrad/UKBinCollectionData#1013] [Q&A] [answered]
I am trying to set the bin colours and Icon using the example JSON code provided in the README. I am pretty sure it is me being a bit thick, but when I enter the following into the configuration wizard I get an 'Invalid JSON format' message

```
{
  "Refuse Collection Service": 
  {
    "icon": "mdi:trash-can",         
    "color": "black"     
  }
}
{
  "Kerbside Recycling Collection Service": 
  {
    "icon": "mdi:recycle",         
    "color": "green"     
  }
}
```

Any assistance would be most helpful. Thanks.

**Comment by m26dvd** [comment, marked as answer]
Try the following

{
  "Refuse Collection Service": 
  {
    "icon": "mdi:trash-can",         
    "color": "black"     
  },
  "Kerbside Recycling Collection Service": 
  {
    "icon": "mdi:recycle",         
    "color": "green"     
  }
}

**Reply by ciderwarehouse1** [reply]
Thanks @m26dvd , that worked a treat.

Thank you also to @robbrad for providing this fab add-in.

**Reply by robbrad** [reply]
Kudos to @m26dvd
